### DISCUSSION TITLE
How can i disable update check?

### DISCUSSION THREAD

**Discussion by Flashwalker** [TriliumNext/Trilium#2999] [Q&A] [answered]
How do I turn off the update check? And all other unwanted network connections?

**Comment by sigaloid** [comment, marked as answer]
Hi, I've submitted a PR to add this feature: https://github.com/zadam/trilium/pull/3000

This disables specifically the update check, as far as I know there's no other network connections besides the UI to the server

**Reply by zadam** [reply]
Merged, it's now possible to configure in Options -> Other:

![image](https://user-images.githubusercontent.com/617641/179857455-64572dae-2d3f-4af7-bf2f-2f290f7add8a.png)

@sigaloid is right, there's no other network communication.
